DataFormats/HcalRecHit/src/HcalRecHitDump_cfg.py

```diff
@@ -0,0 +1,50 @@
+# Run with something like the following:
```

Comment:
we should not have py files in /src directories
there is a /python directory for that
... _cfg files usually live in /test